src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,17 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+import Todo from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={`TodoInfo ${todo.completed === true ? 'TodoInfo--completed' : ''}`}
```

Comment:
To avoid mistakes and make your code more readable in the future I strongly recommend you to use classNames in the same cases.
And in this case, if you write todo.completed it means it's already true, so you don't need to compare it with true.
className={classNames('TodoInfo', {
      'TodoInfo--completed': todo.completed,
    })}